notebooks/official/automl/sdk_automl_tabular_regression_batch_bq.ipynb

```diff
@@ -0,0 +1,969 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.            "SELECT * FROM <code>python-docs-samples-tests.gsod_results_20220227043706.*</code>"
</pre><p>Is this always the dataset name?</p><p> </p><p>Or do you want to get the output from the job to populate it?</p><p> </p><p>As mentioned before, you could use the BQ project variable here too.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/331/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
Oh good catch, I must have left it there during testing.